static/src/javascripts/es6/projects/common/modules/crosswords/clues.js

```diff
@@ -1,9 +1,10 @@
 /* jshint newcap: false */
+import classNames from 'classnames';
+import React from 'react';
+
 import _ from 'common/utils/_';
-import React from 'react/addons';
 
-var classSet = React.addons.classSet,
-    Clue = React.createClass({
+var Clue = React.createClass({
```

Comment:
Yeah, you could argue even further, that given ES6 classes have no sensible 'this' autobind, and no mixin/multi inheritance, that it is fair to expect the same situation with React. That would green light using class extension with React for me.

I just see a future codebase littered with function binds everywhere, and React seems to have a nice solution, seems a shame to lose it.